.github/workflows/ci-stable-release.yml

```diff
@@ -43,10 +43,19 @@ jobs:
           regctl image copy kubearmor/kubearmor-operator:$STABLE_VERSION kubearmor/kubearmor-operator:stable --digest-tags
           regctl image copy kubearmor/kubearmor-snitch:$STABLE_VERSION kubearmor/kubearmor-snitch:stable --digest-tags
 
+      - name: Publish Helm chart
+        env:
+          # Access token which can push to a different repo in the same org
+          GH_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+        run: |
+          STABLE_VERSION=`cat STABLE-RELEASE`
+          gh release create --repo daemon1024/charts $STABLE_VERSION --generate-notes
```

Comment:
`kubearmor/charts`

Reply:
This will only create a release with stable tag that'll contain source code. 

Do we also plan to add a workflow in kubearmor/charts repo where we populate the new chart version once the release is created? Is this how the full workflow to updating `.tgz` will look like. Mostly think about how the following will be handled which was previously done via action extension.

```yaml
      - name: Publish Helm chart
        uses: delusionaloptimist/helm-gh-pages@master
        with:
          # Access token which can push to a different repo in the same org
          token: ${{ secrets.GH_ACCESS_TOKEN }}
          charts_dir: deployments/helm
          # repo where charts would be published
          owner: kubearmor
          repository: charts
          branch: gh-pages
          charts_url: https://kubearmor.github.io/charts
          commit_username: "github-actions[bot]"
          commit_email: "github-actions[bot]@users.noreply.github.com"
```

Reply:
@kranurag7 We already have the workflow in charts repo https://github.com/kubearmor/charts/blob/gh-pages/.github/workflows/release.yaml